Error at make

alejandro@alejandro-Gigabyte:~/p2pool$ make
mkdir -p "/home/alejandro/p2pool/.cache"/pyenv
curl -L 'https://pypi.python.org/packages/source/v/virtualenv/virtualenv-1.11.6.tar.gz' >'/home/alejandro/p2pool/.cache/pyenv/virtualenv-1.11.6.tar.gz' || { rm -f '/home/alejandro/p2pool/.cache/pyenv/virtualenv-1.11.6.tar.gz'; exit 1; }
/bin/sh: 1: curl: not found
Makefile:51: recipe for target '/home/alejandro/p2pool/.cache/pyenv/virtualenv-1.11.6.tar.gz' failed
make: *** [/home/alejandro/p2pool/.cache/pyenv/virtualenv-1.11.6.tar.gz] Error 1

--- a/index.html
+++ b/index.html
@@ -376,7 +376,6 @@ brew install git
 <pre>
 git clone https://github.com/forrestv/p2pool.git
 cd p2pool
-make
 </pre>
   <p>The following packages are required, and will be automatically installed when you run 'make':</p>
   <ul>
@@ -429,7 +428,7 @@ rpcpassword=<span id="password">(random data)</span>
   </div>
 
   <h3><span class="num">2.</span> Run P2Pool</h3>
-  <p><a href="#download">Download P2Pool</a> and execute run_p2pool.exe (Windows) or 'make run' (Linux/OS X/source).</p>
+  <p><a href="#download">Download P2Pool</a> and execute run_p2pool.exe (Windows) or 'python run_p2pool.py' (Linux/OS X/source).</p>
   <p>P2Pool will start downloading the sharechain, displaying sections like this occasionally:</p>
 <pre>
 2014-01-09 17:15:45.075550 P2Pool: 0 shares in chain (0 verified/3133 total) Peers: 5 (0 incoming)

